crates/l2/sequencer/l1_committer.rs

```diff
@@ -167,21 +168,20 @@ async fn commit_next_batch_to_l1(state: &mut CommitterState) -> Result<(), Commi
     let batch = match state.rollup_store.get_batch(batch_to_commit).await? {
         Some(batch) => batch,
         None => {
-            let last_committed_batch = state
-                .rollup_store
-                .get_batch(last_committed_batch_number)
-                .await?.ok_or(CommitterError::InternalError(format!("Failed to get batch with batch number {last_committed_batch_number}. Batch is missing when it should be present. This is a bug")))?;
-            let first_block_to_commit = last_committed_batch.last_block + 1;
+            let last_committed_blocks = state.rollup_store.get_block_numbers_by_batch(last_committed_batch_number).await?.ok_or(CommitterError::InternalError(format!("Failed to get batch with batch number {last_committed_batch_number}. Batch is missing when it should be present. This is a bug")))?;
+            let last_block = last_committed_blocks.last().unwrap();
```

Comment:
Is this safe?

Reply:
It technically is but it's better to turn it into an internal error (basically the same way it's done in the line above the unwrap)